website/docs/r/mks_cluster_v1.html.markdown

```diff
@@ -82,6 +82,14 @@ resource "selectel_mks_cluster_v1" "basic_cluster" {
   
   * `true` - Kube API is available only from the cluster network.
 
+* `enable_audit_logs` - (Optional) Specifies if audit logs should be collected. Learn how to [configure export of audit logs to the log storage and analysis system](https://docs.selectel.ru/en/cloud/managed-kubernetes/clusters/logs/#configure-export-of-audit-logs).
```

Comment:
I suggest that we correct the sentence like this:
 Enables or disables collection of audit logs. Learn how to [configure export of audit logs to a logging system](https://docs.selectel.ru/en/cloud/managed-kubernetes/clusters/logs/#configure-export-of-audit-logs).